package.json

```diff
@@ -23,6 +23,8 @@
     "clean": "rm -rf dist bundle.* index.html && mkdir dist",
     "build": "npm run clean && babel src -d dist --copy-files && node-sass src/main.scss main.css",
     "lint": "eslint src tests showcase --ignore-pattern node_modules --ignore-pattern bundle.js",
+    "lint-styles": "stylelint examples/*.scss showcase/*.scss src/*.scss src/styles/*.scss --syntax scss",
+    "lint-styles-fix": "stylefmt -r examples/*.scss showcase/*.scss src/*.scss src/styles/*.scss --syntax scss",
```

Comment:
Lot of repetition here, can't we avoid that? Also unsure if we need to add `stylefmt`, it's useful the first time, but not so much after.